api/core/v1alpha1/clustercatalog_types.go

```diff
@@ -129,11 +129,9 @@ type ClusterCatalogStatus struct {
 	//  type: Image
 	// +optional
 	ResolvedSource *ResolvedCatalogSource `json:"resolvedSource,omitempty"`
-	// contentURL is a cluster-internal URL from which on-cluster components
-	// can read the content of a catalog
+	// baseURL is a cluster-internal URL from which on-cluster components can access the API endpoint for this catalog
```

Comment:
Not sure of the "how", but it would be really good if we left some breadcrumbs in the GoDoc to direct someone to more information about what paths/APIs are available under this base URL.

Perhaps for now, we could just directly talk about `v1/all`, and then in the future when we add a swagger/OpenAPI endpoint, we could update the GoDoc to direct folks there?

```suggestion
	// baseURL is a cluster-internal base URL from which on-cluster components can access the API endpoint for this catalog
```

Reply:
I was trying to prevent putting any v1 API specific details in the CRD & derived docs, since this is ostensibly the generic endpoint.